### ISSUE TITLE
Code styles aren't used in Typlog because `code` and `tt` aren't direct children of `p`

### ISSUE THREAD

**Issue by michaellenaghan** [typlog/yue.css#20]
https://github.com/typlog/yue.css/blob/76f4cca3898c206fd2e0fcda7431ffecc959046e/src/yue.css#L143-L152

That isn't really a `yue.css` problem, but maybe it's unexpected. And maybe it implies that this is over-specified?

**Comment by lepture** [comment]
@michaellenaghan `code` is. `tt` was used for sphinx.

**Comment by michaellenaghan** [comment]
I'm not sure if "`code` is" means "`code` is a direct descendent of `p`" or "`code` is unexpected" or "`code` is over-specified" or... something else?

Just to make sure we're on the same page, this is the structure I'm seeing in Typlog:

```html
<p>
    ...
</p>
<div class="block-code" ...>
    <div class="highlight" ...>
        <pre>
            <span>
            </span>
            <code>
                ...
            </code>
        </pre>
    </div>
</div>
<p>
    ...
</p>
```

Again, not actually a `yue.css` problem, but maybe unexpected.

**Comment by lepture** [comment]
Try with

```
This is `code`
```

**Comment by michaellenaghan** [comment]
Ah, sorry, yes; I know that.

I looked into this because, in Nezu, the font size in code listings is 18px, which looks... very big. 

I noticed that some themes specifically reduce the font size in code listings. Maybe Nezu should too? I do in my variant. 

But I thought that maybe themes had to reduce the font size because this bit of CSS was meant to be triggered, but wasn't? That's why I logged the issue here and not in Nezu.

At this point I think you're saying this "works as designed," so I'll close it.
